IPython/core/tests/test_alias.py

```diff
@@ -38,4 +38,25 @@ def test_alias_args_error():
     with capture_output() as cap:
         _ip.run_cell('parts 1')
 
-    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
\ No newline at end of file
+    nt.assert_equal(cap.stderr.split(':')[0], 'UsageError')
+    
+def test_alias_args_commented():
+    """Check that alias correctly ignores 'commented out' args"""
+    _ip.magic('alias commetarg echo this is %%s a commented out arg')
+    
+    with capture_output() as cap:
+        _ip.run_cell('commetarg')
```

Comment:
Yeah, it is an abbreviation of "comment met (Dutch for "with") argument".  There is no rational reason why I didn't just call it 'x'.  It would have been a lot easier, and as you pointed out, it's a test so it doesn't matter that it isn't a "meaningful variable name".  I too often have no idea why I do the stupid things I do.